packages/open-next/src/core/routing/util.ts

```diff
@@ -105,9 +105,14 @@ export function convertRes(res: OpenNextNodeResponse): InternalResult {
   const isBase64Encoded =
     isBinaryContentType(headers["content-type"]) ||
     !!headers["content-encoding"];
-  // We cannot convert the OpenNextNodeResponse to a ReadableStream directly
-  // You can look in the `aws-lambda.ts` file for some context
-  const body = Readable.toWeb(Readable.from(res.getBody()));
+  const body =
+    res._chunks == null
+      ? new ReadableStream({
+          start(controller) {
+            controller.close();
+          },
+        })
+      : ReadableStream.from(res._chunks);
```

Comment:
Everything in this repo is targeting Node 18 (including all esbuild configs and typescript configs), so I suspect this may require a major bump.

Reply:
Question tho... if this is targeting 18.x... is CI not targeting that?